app/src/main/res/layout/fragment_marketplace.xml

```diff
@@ -34,14 +35,6 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toTopOf="parent" />
 
-            <ImageView
-                android:layout_width="14dp"
-                android:layout_height="14dp"
-                android:layout_marginVertical="8dp"
-                android:layout_marginStart="10dp"
-                android:src="@drawable/icon_down_arrow"
-                app:layout_constraintStart_toEndOf="@id/country_tx"
-                app:layout_constraintTop_toTopOf="@id/country_tx" />
 
             <ImageView
                 android:id="@+id/plus_imageview"
```

Comment:
also at search_imageview ->Where's the ripple effect on the image button?

Reply:
👍